pkg/capacity/options.go

```diff
@@ -7,6 +7,8 @@ type Options struct {
 	ShowPods              bool
 	ShowUtil              bool
 	ShowPodCount          bool
+	HideRequests         bool
+	HideLimits           bool
```

Comment:
Mind updating the spacing to match the other lines?

Reply:
oops